conf/deployment/aws/ipi_3az_rhcos_3m_3w_3i_vault_v2.yaml

```diff
@@ -0,0 +1,37 @@
+---
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+  kms_deployment: true
+  ocs_operator_nodes_to_label: 0
+ENV_DATA:
+  platform: 'aws'
+  deployment_type: 'ipi'
+  region: 'us-east-2'
+  worker_availability_zones:
+    - 'us-east-2a'
+    - 'us-east-2b'
+    - 'us-east-2c'
+  infra_availability_zones:
+    - 'us-east-2a'
+    - 'us-east-2b'
+    - 'us-east-2c'
+  master_availability_zones:
+    - 'us-east-2a'
+    - 'us-east-2b'
+    - 'us-east-2c'
+  worker_replicas: 3
+  master_replicas: 3
+  worker_instance_type: 'm5.4xlarge'
+  encryption_at_rest: true
+  vault_deploy_mode: external
+  use_vault_namespace: false
+  KMS_PROVIDER: vault
+  KMS_SERVICE_NAME: vault
+  VAULT_CACERT: "ocs-kms-ca-secret"
+  VAULT_CLIENT_CERT: "ocs-kms-client-cert"
+  VAULT_CLIENT_KEY: "ocs-kms-client-key"
+  VAULT_SKIP_VERIFY: false
+  VAULT_BACKEND: "v2"
+REPORTING:
+  polarion:
+    deployment_id: 'OCS-2430'
```

Comment:
Do we want the same polarion-id as the config without vault?

Reply:
Discussed with Elad and he is fine to keep this same for 3m3w3i poloarion ID